### ISSUE TITLE
AppliedSuccefully doesn't set condition properly

### ISSUE THREAD

**Issue by k0da** [weaveworks/tf-controller#436]
``` 
 - lastTransitionTime: "2022-10-26T10:35:55Z"
    message: 'Applied successfully: awesome-repo/f368f993dc74ca0f5a6588e24b356f518c8b7ce8'
    reason: TerraformAppliedSucceed
    status: Unknown
    type: Ready
```
status should be True

**Comment by k0da** [comment]
https://github.com/weaveworks/tf-controller/blob/main/api/v1alpha1/terraform_types.go#L550
explicitly sets unknown

**Comment by chanwit** [comment]
Hi @k0da 
Thank you for that.

We're working on fixing this and other related issues in a different PR.

**Comment by chanwit** [comment]
Fixed by #438 and #440 (v0.13.0)
Please feel free to re-open it if you still see the error.
